<!--xml-->
<launch>
   <param name="target_count" value="2" />
  <node name="pick_targets_from_train_images" pkg="icarus_rover_rc" type="pick_targets_from_train_images" output="screen">
   
  </node>
</launch>
